eventq_rabbitmq/lib/eventq_rabbitmq/rabbitmq_queue_worker.rb

```diff
@@ -327,6 +324,11 @@ def configure(queue, options = {})
 
       private
 
+      def new_client_instance(options)
+        raise "[#{self.class}] - MQ Endpoint not present." unless options[:mq_endpoint]
+        EventQ::RabbitMq::QueueClient.new({endpoint: options[:mq_endpoint] })
```

Comment:
this option key, `:mq_endpoint` is not used in the constructor.  You need to fix this as well
https://github.com/Sage/eventq/blob/cb70cde14ec5b28119167ce0d21ad21d1defe3dd/eventq_rabbitmq/lib/eventq_rabbitmq/rabbitmq_queue_client.rb#L9-L13